appengine/.gcloudignore

```diff
@@ -2,14 +2,12 @@
 .*
 *.soy
 *.komodoproject
-/deploy
+deploy
 /static/appengine/
-/static/closure/
 /static/demos/plane/soy/*.jar
 /static/demos/plane/xlf/
 /static/externs/
 /static/msg/json/
-/static/node_modules/
```

Comment:
Why remove the git ignore for the node_modules?

Reply:
The files copied using the command `git archive HEAD` are those committed to the repository and do not include the node_modules directory (removing it reduces confusion since these files don't exist to be ignored in the first place).